samples/graph-rsc/nodeJs/teamsManifest/manifest.json

```diff
@@ -0,0 +1,61 @@
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.8/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.8",
+  "version": "1.0.0",
+  "id": "f0a1c0e8-92a5-4519-a67d-5ffe54293cbf",
```

Comment:
Could you please remove actual values for id, configurationUrl & webApplicationInfo.Id and put placeholders?

Reply:
removed